src/functionalTest/groovy/com/github/spotbugs/snom/BasePluginFunctionalTest.groovy

```diff
@@ -123,4 +123,42 @@ task spotbugsMain(type: com.github.spotbugs.snom.SpotBugsTask) {
         File report = rootDir.toPath().resolve("build").resolve("reports").resolve("spotbugs").resolve("main.xml").toFile()
         !report.isFile()
     }
+
+    @Unroll
+    def 'shows suggestion to enable report when failures are found (Worker API? #isWorkerApi)'() {
+        given:
+        def badCode = new File(rootDir, 'src/main/java/Bar.java')
+        badCode << '''
+        |public class Bar {
+        |  public int unreadField = 42; // warning: URF_UNREAD_FIELD
+        |}
+        |'''.stripMargin()
+        buildFile << """
+task spotbugsMain(type: com.github.spotbugs.snom.SpotBugsTask) {
+    dependsOn 'classes'
+    classDirs = sourceSets.main.output
+}
+"""
+        when:
+        def arguments = [':spotbugsMain']
+        if(!isWorkerApi) {
+            arguments.add('-Pcom.github.spotbugs.snom.worker=false')
```

Comment:
maybe add a comment on why we need to turn off the worker api? Just looking at the code it isn't obvious to me.

Reply:
added a comment, please check 42802b6